make-dist

```diff
@@ -85,13 +85,33 @@ _python_autoselect() {
 build_dashboard_frontend() {
   CURR_DIR=`pwd`
   TEMP_DIR=`mktemp -d`
+
   $CURR_DIR/src/tools/setup-virtualenv.sh --python=$(_python_autoselect) $TEMP_DIR
   $TEMP_DIR/bin/pip install nodeenv
   $TEMP_DIR/bin/nodeenv -p --node=10.16.0
   cd src/pybind/mgr/dashboard/frontend
+
+  DEFAULT_LANG=`cat package.json | jq -r .config.locale`
```

Comment:
If npm is available, this could be easily retrieved with `npm config get ceph-dashboard:locale`.